torchvision/csrc/models/resnet.h

```diff
@@ -126,7 +126,7 @@ ResNetImpl<Block>::ResNetImpl(
 
   for (auto& module : modules(/*include_self=*/false)) {
     if (auto M = dynamic_cast<torch::nn::Conv2dImpl*>(module.get())) {
-      torch::nn::init::xavier_normal_(M->weight); // TODO kaiming
+      torch::nn::init::kaiming_normal_(M->weight);
```

Comment:
Same comment here about the arguments
https://github.com/pytorch/vision/blob/f75b8148e03cc3155aff6431cd785ea02f299420/torchvision/models/resnet.py#L118